src/test/ui/attributes/issue-48638.rs

```diff
@@ -0,0 +1,21 @@
+// check-pass
```

Comment:
`attributes/` is probably not the best place since the ICE was coming from wfcheck.

Reply:
So, should be in `repr`?

Reply:
Probably? or something more WF-related. :)

Reply:
Okay, I placed it in `wf/`.